src/spikeinterface/core/sortinganalyzer.py

```diff
@@ -659,11 +642,12 @@ def create_zarr(cls, folder, sorting, recording, sparsity, return_scaled, rec_at
     @classmethod
     def load_from_zarr(cls, folder, recording=None, backend_options=None):
         import zarr
+        from .loading import load
 
         backend_options = {} if backend_options is None else backend_options
         storage_options = backend_options.get("storage_options", {})
 
-        zarr_root = zarr.open_consolidated(str(folder), mode="r", storage_options=storage_options)
+        zarr_root = super_zarr_open(str(folder), mode="r", storage_options=storage_options)
```

Comment:
@alejoe91  : Here, is it a problem that the zarr_root could be eventually not consolidated ?

Reply:
It could for some older assets. Plus in append mode we can't use consolidated